llvm/include/llvm/BinaryFormat/Dwarf.h

```diff
@@ -616,20 +616,20 @@ enum AcceleratorTable {
 // Uniquify the string hashes and calculate the bucket count for the
 // DWARF v5 Accelerator Table. NOTE: This function effectively consumes the
 // 'hashes' input parameter.
-inline uint32_t getDebugNamesBucketCount(MutableArrayRef<uint32_t> hashes,
-                                         uint32_t &uniqueHashCount) {
-  uint32_t BucketCount = 0;
+inline std::pair<uint32_t, uint32_t>
+getDebugNamesBucketAndHashCount(MutableArrayRef<uint32_t> hashes) {
+  uint32_t bucketCount = 0;
```

Comment:
Why did you change the case? Doesn't LLVM use UperCamelCase for variables?

Reply:
good point, I had missed this

Reply:
I thought LLVM variables were supposed to start with lower case?  So I thought I was fixing a problem? I can switch it back if you think I should...

Reply:
You're probably thinking of functions, which (unlink LLDB) use `lowerCamelCase`: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
Thanks! Done.